proto/desmos/profiles/v3/msgs_app_links.proto

```diff
@@ -11,9 +11,6 @@ import "ibc/core/client/v1/client.proto";
 // MsgLinkApplication defines a msg to connect a profile with a
 // centralized application account (eg. Twitter, GitHub, etc).
 message MsgLinkApplication {
-  option (gogoproto.equal) = false;
-  option (gogoproto.goproto_getters) = false;
```

Comment:
All of these have been removed since they are not only useless, but also prevented some of the messages to implement `SubspaceMsg` correctly.